lib/jekyll/version.rb

```diff
@@ -1,3 +1,3 @@
 module Jekyll
-  VERSION = '3.1.6'
+  VERSION = "3.1.6".freeze
```

Comment:
Anything to do with freezing strings can be skipped, there are plans to add the magic comments soon.